src/test/java/emissary/output/DropOffUtilTest.java

```diff
@@ -601,6 +602,41 @@ void testExtractUniqueFileExtensions() {
         assertTrue(fileExts.contains("mp3"), "FILEXT values should contain \"mp3\"");
     }
 
+    @Test
+    void testExtractFullFilepaths() {
```

Comment:
Method name doesn't match the logic or name of the method-under-test (`extractUniqueFileExtensions`)